schemas/telemetry/account-ecosystem/account-ecosystem.4.schema.json

```diff
@@ -1,5 +1,9 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "mozPipelineMetadata": {
+    "bq_dataset": "account_ecosystem",
+    "bq_table": "desktop"
```

Comment:
In this case, the bq fields _were_ defined in the template, so those custom values appear here rather than the defaults (which would have been "telemetry" and "account_ecosystem_v4").